### ISSUE TITLE
rabbitmq_user.present broken on Ubuntu 16.04 Xenial

### ISSUE THREAD

**Issue by edgan** [saltstack/salt#35003]
### Description of Issue/Question
  I am trying to create a rabbitmq user on Ubuntu 16.04 Xenial via salt-ssh. It tries to run "rabbitmq list_users" as the rabbitmq user. It needs to be the rabbitmq user to work. It fails because the PATH set doesn't include /usr/sbin. cmd.run has env to be able to set the PATH, but rabbitmq_user.present doesn't.

  My workaround is to symlink ../sbin/rabbitmqctl to /usr/bin/rabbitmqctl.

### Setup
```
rabbitmq_user_admin:
  rabbitmq_user.present:
    - name: {{ rabbitmq.dbusers.admin.username }}
    - password: {{ rabbitmq.dbusers.admin.password }}
    - tags:
      - administrator
    - perms:
      - '/':
        - '.*'
        - '.*'
        - '.*'
    - runas: rabbitmq

----------
          ID: rabbitmq_user_admin
    Function: rabbitmq_user.present
        Name: admin
      Result: False
     Comment: Error: Unable to run command '['rabbitmqctl', 'list_users']' with the context '{'shell': False, 'stderr': -2, 'stdout': -1, 'bg': False, 'with_communicate': True, 'preexec_fn': <functools.partial object at 0x7f2c9ade7628>, 'env': {'LC_NUMERIC': 'C', 'LC_CTYPE': 'C', 'LOGNAME': 'rabbitmq', 'USER': 'rabbitmq', 'HOME': '/var/lib/rabbitmq', 'LC_PAPER': 'C', 'PATH': '/usr/local/bin:/usr/bin:/bin:/usr/local/games:/usr/games'
```

### Versions Report
2016.3.1

**Comment by Ch3LL** [comment]
@edgan I think it would be a good idea to add the ability to use `env` with the `rabbitmq` module as well. I'll approve as a feature request. Thanks.

Also i wonder if another workaround would be to use the environ state alongside your state file as documented [here](https://docs.saltstack.com/en/latest/ref/states/all/salt.states.environ.html)

**Comment by cachedout** [comment]
Closed via #35232
